### PR TITLE
Ggoryl final approach

### PR DESCRIPTION
**Pull request by polquant**

### CODE REVIEW

File: models/model.py [hunk 1]
```diff
@@ -241,9 +237,12 @@ def validation_epoch_end(self, out):
     def configure_optimizers(self):
         optimizer = torch.optim.RAdam(self.parameters(), lr=self.lr)
 
-        steps_per_epoch = self.steps_per_epoch // self.trainer.accumulate_grad_batches
```
Comment (polquant) [inline models/model.py:244]: [Bug] for `acumulate_grad_batches` > 0 scheduler has been finishing to quickly with an exception

**Review by wiktorfilipiuk — APPROVED** [review]

**Review by AgaMiko — APPROVED** [review]
